yt/yt/client/table_client/wire_protocol.h

```diff
@@ -280,13 +280,24 @@ struct IWireProtocolReader
 
 ////////////////////////////////////////////////////////////////////////////////
 
+struct TWireProtocolOptions {
+    ui32 MaxStringValueLength = NYT::NTableClient::MaxStringValueLength;
+    ui32 MaxAnyValueLength = NYT::NTableClient::MaxAnyValueLength;
+    ui32 MaxCompositeValueLength = NYT::NTableClient::MaxCompositeValueLength;
+    ui32 MaxTimestampCountPerRow = NYT::NTableClient::MaxTimestampCountPerRow;
```

Comment:
Why do we need last 2 fields here?

Reply:
We (In QueryTracker) probably don't. At least we didn't get a case where they matter.

Other wire protocol users may need them, as they may work with different type of data.

>This PR has 2 commits:
>
>1. Adding configurable limits to wire protocol reader
>2. Adding QueryTracker tests and configure limits for wire protocol reader used in QueryTracker
>
>First commit has a separate PR: https://github.com/ytsaurus/ytsaurus/pull/943

Reply:
It seems strange that we specify only these five parameters instead of configuring only the required ones or all
https://github.com/ytsaurus/ytsaurus/blob/ccc0a106bda90562675784da9492e08ec5997c3e/yt/yt/client/table_client/public.h#L93

Reply:
Not all of them are used in wire protocol
We configure everything that is used in wire protocol (`wire_protocol.cpp`) and may require adjusting

Or is there anything else that may need to be configured?